distribution/lib/Standard/Test/0.0.0-dev/src/Test.enso

```diff
@@ -226,7 +226,9 @@ type Test
         need to be skipped will not change between versions, so this method should be
         used carefully.
     get_source_location skip_frames:Integer -> Text =
-        stack = Standard.Base.Runtime.get_stack_trace
-        element = stack . at skip_frames+1
-        loc = element . source_location
-        loc . to_display_text
+        # stack = Standard.Base.Runtime.get_stack_trace
+        # element = stack . at skip_frames+1
+        # loc = element . source_location
+        # loc . to_display_text
+        skip_frames.to_text
+        "<disabled>"
```

Comment:
```suggestion
        case Environment.get "ENSO_INCLUDE_STACK_TRACE" of
            Nothing ->
                _ = skip_frames
                "<disabled>"
            _ ->
                stack = Standard.Base.Runtime.get_stack_trace
                element = stack . at skip_frames+1
                loc = element . source_location
                loc . to_display_text
```


You may not rely on the traces as much but Library devs rely on them A LOT.

Let's at least hide this behind an env var instead of disabling completely. What do you think? Disabled by default so you get fast (but less useful as no traces...) CI, but at least locally we can still work.